mpmp/prediction/classification.py

```diff
@@ -395,14 +416,16 @@ def get_preds(X_test_df, y_test_df, cv_pipeline, fold_no):
 
     Also returns true class, to enable quantitative comparisons in analyses.
     """
-
     # get probability of belonging to positive class
     y_scores_test = cv_pipeline.decision_function(X_test_df)
     y_probs_test = cv_pipeline.predict_proba(X_test_df)
 
-    # make sure we're actually looking at positive class prob
-    assert np.array_equal(cv_pipeline.best_estimator_.classes_,
-                          np.array([0, 1]))
+    try:
+        # make sure we're actually looking at positive class prob
+        assert np.array_equal(cv_pipeline.best_estimator_.classes_,
+                              np.array([0, 1]))
+    except AttributeError:
+        assert np.array_equal(cv_pipeline.classes_, np.array([0, 1]))
```

Comment:
I'm not sure when the `best_estimator_` attribute is expected to exist, but it may be clearer to check for it explicitly rather than by catching an AttributeError

Reply:
Makes sense - I was trying to go for [Python-style EAFP rather than LBYL](https://docs.python.org/3.8/glossary.html#term-eafp), but I agree that a comment would probably make things a bit clearer here.